### PR TITLE
DRY up Orangelight playbook and variables

### PR DESCRIPTION
**Pull request by maxkadel**
- Pull out shared variables into `common.yml` file
- Alphabetize variable names for ease of finding values
- Use single playbook which defaults to running on staging, can add `-e runtime_env={environment}` to run on production or qa. 
- Take figgy url out of vault, since it's all over this repo anyway

### CODE REVIEW

File: group_vars/orangelight/staging.yml [hunk 1]
```diff
@@ -1,132 +1,33 @@
...
-    value: '{{ vault_clancy_api_base_url }}'
-  - name: BIBDATA_BASE
-    value: 'https://bibdata-staging.princeton.edu'
-  - name: ORANGELIGHT_ADMIN_NETIDS
-    value: '"{{ ol_admin_netids | join(" ") }}"'
-sneakers_workers: EventHandler
-sneakers_worker_name: orangelight-sneakers
-ol_rails_solr_url: "{{ ol_staging_solr_url }}"
-voyager_ub_id: "{{ omit }}"
+reserves_core: "reserves"
```
Comment (sandbergja) [inline group_vars/orangelight/staging.yml:29]: @christinach can we just remove this variable (and from the other environments too)?

File: group_vars/orangelight/common.yml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    value: '"{{ ol_admin_netids | join(" ") }}"'
+  - name: SCSB_AUTH_KEY
+    value: "{{ scsb_auth_key }}"
+  - name: SMTP_HOST
+    value: "{{ ol_smtp_host | default('localhost') }}"
+  - name: SMTP_PORT
+    value: "{{ ol_smtp_port | default('1025') }}"
+  - name: SOLR_URL
+    value: "{{ ol_solr_url }}"
+  - name: voyager_ub_id
```
Comment (sandbergja) [inline group_vars/orangelight/common.yml:107]: @christinach can we remove `voyager_ub_id`?

**Review by sandbergja — COMMENTED** [review]
Looks good; thanks for doing this @maxkadel !  I'm wondering if we can do a little additional cleanup as part of this PR, but it's not a blocker.

File: group_vars/orangelight/common.yml (hunk 2 above)
Comment (christinach) [reply group_vars/orangelight/common.yml:107]: yes we can

File: group_vars/orangelight/staging.yml (hunk 1 above)
Comment (christinach) [reply group_vars/orangelight/staging.yml:29]: yes we can remove this

**Review by sandbergja — APPROVED** [review]
Thanks @maxkadel !

**Comment by maxkadel** [comment]
Thanks @sandbergja! Sorry I didn't see your review earlier!

**Pull request merged**